### PR TITLE
VBET Evidence, Normalized, & Combined Evidence Symbology

### PR DESCRIPTION
**Pull request by shelbysawyer**
This includes symbology key and .lyr file additions for VBET.

### CODE REVIEW

File: RaveBusinessLogic/VBET.xml [hunk 1]
```diff
@@ -113,8 +113,8 @@
           </Repeater>
           <Node label="Combined Evidence Rasters">
             <Children>
-              <Node label="Channel Evidence" xpath="Intermediates/Raster[@id='EVIDENCE_CHANNEL']" type="raster" symbology="" transparency="40" />
-              <Node label="Topographic Evidence" xpath="Intermediates/Raster[@id='EVIDENCE_TOPO']" type="raster" symbology="" transparency="40" />
+              <Node label="Channel Evidence" xpath="Intermediates/Raster[@id='EVIDENCE_CHANNEL']" type="raster" symbology="channelevidence" transparency="40" />
+              <Node label="Topographic Evidence" xpath="Intermediates/Raster[@id='EVIDENCE_TOPO']" type="raster" symbology="topoevidence" transparency="40" />
             </Children>
```
Comment (joewheaton) [inline RaveBusinessLogic/VBET.xml:118]: Does this collide with or conflict with other vbet_symbology business logic pull request (#118)  I just approved? Please avoid putting pull requests in simultaneously on different branches that make changes to same files.

**Review by joewheaton — CHANGES_REQUESTED** [review]
I am not going to review these binary files by downloading them. Please provide me screen shots of what each layer file is going to look like. They can just be a comment in pull request. Maybe sperate them using the `## Normalized Channel Layer` notation for headings to make it easy to see. Thanks! Also, I am thinking that the businessLogic.xml commit will collide with your other pull request.

**Comment by MattReimer** [comment]
@shelbysawyer what's the status here? Last activity was 23 days ago.

**Comment by shelbysawyer** [comment]
@MattReimer Sorry for the delay. I've been out of office since 7/12 and am getting caught up on everything today. I'll get this closed by end of week.

**Comment by MattReimer** [comment]
> @MattReimer Sorry for the delay. I've been out of office since 7/12 and am getting caught up on everything today. I'll get this closed by end of week.

No worries and no rush. I'm just trying to figure out if any issues are stale. Sometimes github issues fall through the cracks.

**Comment by joewheaton** [comment]
@MattReimer I suspect @shelbysawyer will get back to this as soon as we have first round of transform curves off her plate.

**Comment by MattReimer** [comment]
@shelby I noticed this got closed but it was never merged. Are you giving up on these changes?

**Pull request closed without merging**